dapp/src/helpers/claims.ts

```diff
@@ -316,6 +353,10 @@ export const claimToOutlink = (ct: ClaimType, c: Claim): string => {
       draft = draft as TwitterDraft;
       return `https://www.twitter.com/${draft.handle}`;
     }
+    case 'dns': {
+      draft = draft as DnsDraft;
+      return formatWebsite(draft.address);
```

Comment:
That doesn't really make sense, you can't assume there's a website for a given domain name. What could be done is send a GET or a HEAD with HTTP or HTTPS, and if there's a response you can replace the website from the self-certification (and add a check mark). Otherwise, it might need a different kind of display

Reply:
Hum but that might not work because of CORS :thinking:

Reply:
We now changed to copy the address on click